config/settings/common.py

```diff
@@ -153,7 +153,10 @@
         # See: https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEMPLATES-BACKEND
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         # See: https://docs.djangoproject.com/en/dev/ref/settings/#template-dirs
-        'DIRS': [],
+        'DIRS': [
+            str(APPS_DIR.path('main/templates')),
```

Comment:
To powinno być automatycznie.

Reply:
To nie jest automatycznie właśnie tzn. aktualnie wyszukuje templatki np. w vendor, a potem w apps dir, a po zmianie najpierw w main/templates, a potem w vendorze, a potem w apps dir. Pozwala to nadpisać templatki.
Poradnia posiada podobna linijkę: https://github.com/watchdogpolska/poradnia/blob/master/config/settings/common.py#L305

Reply:
@mik-laj , bo poradnia nie ma aplikacji main, ale okazuje się, że powinna, jeżeli chce się mieć poprawne tłumaczenia.